content/en/core/overview/db-schema.md

```diff
@@ -178,13 +178,10 @@ All reports:
  - Store the form's identifier in the `form` field
  - May have a `contact` property, which is a minified version of the report author's contact and its hierarchy (see above)
 
-Reports can be about people or places.
+Reports can and should be linked to a contact when possible. The report's associated contact (sometimes called the report's subject) can be either a person or place. The link between report and contact is established by defining one of the following properties within the report:
```

Comment:
> The report's associated contact (sometimes called the report's subject)

I think the text is improved. Thanks for this. 

For the future, I believe this needs further disambiguation, because, historically, there was confusion about what the `contact` really is, and rightly so, since the `contact` field actually references the submitter. I sometimes think we should exclusively use `submitter` and `subject` and try to avoid saying "contact" altogether.